packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -214,36 +214,90 @@
 <section data-test-percy>
   <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
 
-  <p class="dummy-paragraph"><strong>Basic usage:</strong></p>
+  <h4 class="dummy-h4">Tab</h4>
+
+  <h5 class="dummy-h6">Content</h5>
+  <div class="dummy-tab-base-sample">
+    <div>
+      <span class="dummy-text-small">Text only</span>
+      <br />
+      <Hds::Tabs::Tab>Lorem ipsum</Hds::Tabs::Tab>
+    </div>
+    <div>
+      <span class="dummy-text-small">Icon + Text</span>
+      <br />
+      <Hds::Tabs::Tab @icon="hexagon">Lorem ipsum</Hds::Tabs::Tab>
+    </div>
+    <div>
+      <span class="dummy-text-small">Text + Counter</span>
+      <br />
+      <Hds::Tabs::Tab @count="10">Lorem ipsum</Hds::Tabs::Tab>
+    </div>
+    <div>
+      <span class="dummy-text-small">Icon + Text + Counter</span>
+      <br />
+      <Hds::Tabs::Tab @icon="hexagon" @count="10">Lorem ipsum</Hds::Tabs::Tab>
+    </div>
+  </div>
+
+  <h5 class="dummy-h6">States</h5>
+  <div class="dummy-tab-base-sample">
+    {{#each @model.TAG_STATES as |state|}}
+      <div class="dummy-tab-base-sample-item">
+        <span class="dummy-text-small">{{capitalize state}}:</span>
+        <Hds::Tabs::Tab mock-state-value={{state}} mock-state-selector="a">Lorem ipsum</Hds::Tabs::Tab>
+        <Hds::Tabs::Tab @icon="hexagon" @count="10" mock-state-value={{state}} mock-state-selector="a">Lorem ipsum</Hds::Tabs::Tab>
+      </div>
+    {{/each}}
+  </div>
+
+  <h4 class="dummy-h4">Panel</h4>
+  <h5 class="dummy-h6">Content</h5>
+  <Hds::Tabs::Panel><DummyPlaceholder
+      @text="Panel with generic content"
+      @height="50"
+      @background="#eee"
+    /></Hds::Tabs::Panel>
+
+  <h4 class="dummy-h4">Tabs</h4>
+  <h5 class="dummy-h6">Basic usage</h5>
   <Hds::Tabs as |T|>
     <T.Tab>One</T.Tab>
     <T.Tab>Two</T.Tab>
     <T.Tab>Three</T.Tab>
 
-    <T.Panel>Content one <a href="#">Link to test tabbing</a></T.Panel>
-    <T.Panel>Content 2</T.Panel>
-    <T.Panel>Content 3!</T.Panel>
+    <T.Panel><DummyPlaceholder @height="50" @background="#eee"><span>Content one with
```

Comment:
Here I'm using a "standard" placeholder component, like it's used in other showcases (see for example https://github.com/hashicorp/design-system/pull/590)